praetorian_cli/ui/conversation/textual_chat.py

```
@@ -0,0 +1,622 @@
+                        self.update_status("Executing tool...")
+                    elif role == 'tool response':
+                        self.add_tool_message("✅ Tool execution completed")
+                        self.update_status("Tool completed, thinking...")
+                    elif role == 'planner-output':
+                        self.add_system_message("🎯 Processing job completion...")
+                
+                # Update last message key
+                self.last_message_key = messages[-1].get('key', '')
+                
+        except Exception as e:
+            pass
```

Ruff (0.13.1)

`try`-`except`-`pass` detected, consider logging the exception

(S110)